CHANGELOG

```diff
@@ -1,3 +1,8 @@
+7.0.4
+-----
+
+* Bump required shopify_api version to >= 4.2.0.
```

Comment:
I changed this to `4.x` so we don't end up stuck with 4.2.x after 4.3 comes out.